### ISSUE TITLE
High priority - Sub sub (primary forest and introduced) tables should behave exactly the same way- tab 1b

### ISSUE THREAD

**Issue by pengchenglai** [openforis/fra-platform#3147]
@sorja @minotogna 

**Describe the bug**
National Classes from sub sub (**. …of which introduced**) table in ODP view will show the value 0s if sub values (**Plantation forest**) are 0. Not possible to edit empty cells with the value 0s in the second table Plantation forest(example plz see below). 

**To Reproduce**
1. Go to '**Bangladesh** tab 1b'(production)
2. Click on '2015 NDP'
3. Scroll down to 'second table Plantation forest'
4. See error(plz see below)
![Capture](https://github.com/openforis/fra-platform/assets/99216009/c5b7fb5c-3926-4bdd-8bca-b3e3f7fe022c)

**Expected behavior**
Hide National Class from sub sub (**. …of which introduced**) table in ODP view if sub value (**Plantation forest**) is 0.
In other words, sub sub (primary forest and introduced) tables should behave exactly the same way as previously agreed

**Comment by pengchenglai** [comment]
#3103

**Comment by sorja** [comment]
@pengchenglai 
Can you verify if this is fixed in review instance?

Thank you

https://fra-data.review.fao.org/assessments/fra/2025/BGD/originalDataPoints/2015/forestCharacteristics

Note for future (not related to this issue)
- Better check always if the bug exists on review so we avoid working on issues that are already resolved
- Bottom of the page you can see the latest deployment date, if a pull request is merged before this date, it means that it is visible in review instance (also has a last deployment date) but not in production
![image](https://github.com/openforis/fra-platform/assets/5508251/78a4fca0-a5ab-445a-8a1c-8a00a1d3e6b0)

**Comment by pengchenglai** [comment]
@sorja 

Great, it is working for the instance. Thank youuu!

Thanks for reminding me of this! I will definitely check it next time. Actually, I thought there were updates for the platform since the past few weeks. 

Do you know when we will make a new deployment to production? Ideally, there should be one this week at some point, one week before the workshop.

**Comment by sorja** [comment]
@pengchenglai 
We have been working on some larger updates on the platform and this is why we have postponed the deployment to production - to verify the stability on review instance and that everything is working silky smooth!

We normally deploy after a milestone:
https://github.com/openforis/fra-platform/milestone/25

But I think @minotogna will let you know about current deployment schedule when it's time :)

**Comment by minotogna** [comment]
@pengchenglai @sorja we'll deploy tonight or tomorrow, after updating primary forest by climatic domain

**Comment by pengchenglai** [comment]
@minotogna 

Well noted! Thank you:)
